src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -285,12 +382,8 @@ private void updateCAType(Topic topic) {
                 String caType = caTypeList.get(0);
                 certificateManager.update(getPassphrase(), CertificateStore.CAType.valueOf(caType));
             }
-
-            List<String> caCerts = certificateManager.getCACertificates();
-            uploadCoreDeviceCAs(caCerts);
-            updateCACertificateConfig(caCerts);
             updateCaPassphraseConfig(certificateManager.getCaPassPhrase());
-        } catch (KeyStoreException | IOException | CertificateEncodingException | IllegalArgumentException
+        } catch (KeyStoreException | IllegalArgumentException
```

Comment:
There's an existing bug here - if uploading the CA fails, we shouldn't enter a broken state. Should think about how to handle retries

Reply:
We're retrying the cloud request here - https://github.com/aws-greengrass/aws-greengrass-client-device-auth/blob/ff646044c87875f465dba5cccd0b8f5b54132ad1/src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java#L423
Did you mean the same? Also, those exceptions are not from `uploadCoreDeviceCAs`.